src/main/java/org/zone/region/flag/entity/player/interact/itemframe/ItemFrameInteractionListener.java

```diff
@@ -36,9 +37,8 @@ public void onAttack(InteractEntityEvent event, @First Player attacker) {
         Zone zone = opZone.get();
         Group group = zone.getMembers().getGroup(attacker.uniqueId());
         if (group.contains(GroupKeys.INTERACT_ITEMFRAME)) {
-            return;
+            event.setCancelled(true);;
```

Comment:
Why is this edited?

Reply:
shouldn't it be? cuz if the group had that GroupKey the event should be cancelled?

Reply:
It should be cancelled if they dont have the key (as the key is used as a bypass). This worked in 1.0.0 as intended and the code has not been changed since